wp-content/mu-plugins/pantheon/pantheon-page-cache.php

```diff
@@ -92,7 +92,7 @@ protected function setup() {
 
 		add_action( 'admin_post_pantheon_cache_flush_site',  array( $this, 'flush_site' ) );
 
-		if ( ! is_admin() ) {
+        if ( ! is_user_logged_in() ) {
```

Comment:
@stevector we should still check `is_admin()` so that `admin-ajax.php` calls aren't cached.

Something like `if ( ! is_admin() && ! is_user_logged_in() ) {`